lsl/common/stations.py

```diff
@@ -374,26 +351,21 @@ def get_pointing_and_distance(self, locTo):
             Renamed from getPointingAndDirection to get_pointing_and_distance
         """
         
-        ecefFrom = self.geocentric_location
+        ecefFrom = self.earth_location.itrs
         try:
-            ecefTo = locTo.geocentric_location
+            ecefTo = locTo.itrs
         except AttributeError:
-            ecefTo = geo_to_ecef(float(locTo[0])*numpy.pi/180, float(locTo[1])*numpy.pi/180, locTo[2])
+            try:
+                ecefTo = locTo.earth_location
+            except AttributeError:
+                ecefTo = EarthLocation.from_geodetic(locTo[1]*astrounits.deg, locTo[0]*astrounits.deg, height=locTo[2]*astrounits.m,
+                                                     ellipsoid='WGS84')
+            ecefTo = ecefTo.itrs
             
-        ecefFrom = numpy.array(ecefFrom)
-        ecefTo = numpy.array(ecefTo)
-        
-        rho = ecefTo - ecefFrom
-        rot = numpy.array([[ numpy.sin(self.lat)*numpy.cos(self.long), numpy.sin(self.lat)*numpy.sin(self.long), -numpy.cos(self.lat)], 
-                           [-numpy.sin(self.long),                     numpy.cos(self.long),                      0                  ],
-                           [ numpy.cos(self.lat)*numpy.cos(self.long), numpy.cos(self.lat)*numpy.sin(self.long),  numpy.sin(self.lat)]])
-        sez = numpy.dot(rot, rho)
+        aa = AltAz(location=ecefFrom, obstime=ecefTo.obstime, pressure=0)
+        pd = ITRS(ecefTo.cartesian.xyz-ecefFrom.cartesian.xyz, obstime=ecefTo.obstime, location=ecefFrom).transform_to(aa)
         
-        d = numpy.sqrt( (rho**2).sum() )
-        el = numpy.arcsin(sez[2] / d)
-        az = numpy.arctan2(sez[1], -sez[0])
-        
-        return az, el, d
+        return (pd.az.rad, pd.alt.rad, pd.distance.to('m').value)
```

Comment:
This is one area where we need to check to make sure that we are getting identical answers.

Reply:
The distance is at least correct.  I'm not sure about the other two parameters.

Reply:
I guess `get_enz_offset()` vets this since it uses this function internally.